Tests/ScipioKitTests/IntegrationTests.swift

```diff
@@ -154,8 +156,25 @@ final class IntegrationTests: XCTestCase {
             )
 
             for destination in expectedDestinations {
-                let frameworkRoot = xcFrameworkPath
+                let sdkRoot = xcFrameworkPath
                     .appendingPathComponent(destination)
+
+                if let buildOption = buildOptionsMatrix[frameworkName],
+                   buildOption.isDebugSymbolsEmbedded == true,
+                   buildOption.frameworkType == .dynamic {
+                    XCTAssertTrue(
+                        fileManager.fileExists(atPath: sdkRoot
+                            .appendingPathComponent("dSYMs/\(frameworkName).framework.dSYM/Contents/Info.plist").path),
+                        "\(xcFrameworkName) should contain a Info.plist file in dSYMs directory"
+                    )
+                    XCTAssertTrue(
+                        fileManager.fileExists(atPath: sdkRoot
+                            .appendingPathComponent("dSYMs/\(frameworkName).framework.dSYM/Contents/Resources/DWARF/\(frameworkName)").path),
+                        "\(xcFrameworkName) should contain a DWARF file in dSYMs directory"
+                    )
+                }
```

Comment:
Could you check the negative situations?

```
} else {
    XCTAssertFalse(~, "\(xcFrameworkName) should not contain a dSYMs directory")
}

Reply:
sure!

Reply:
@giginet okey, added negative situation assert.